test/module-only-flat.js

```diff
@@ -0,0 +1,82 @@
+
+QUnit.config.reorder = false;
+var tests = [];
+var done = false;
+
+QUnit.testDone( function( details ) {
+	if ( done ) {
+		return;
+	}
+
+	tests.push( {
+		name: details.name,
+		module: details.module,
+		skipped: details.skipped,
+		todo: details.todo
+	} );
+} );
+
+QUnit.done( function() {
```

Comment:
I understand this follows the existing patterns for other tests here, but I've been thinking lately that perhaps we should phase this out since it seems to rely re-opening test suites which is something we actively don't support and afaik already doesn't work in many cases when a JS reporter is used. Incl from the command-line.

Could you see if this would work as a CLI test instead? Then we can simply assert the TAP output instead, which would also make it a more complete assertion for anything unexpected.

Reply:
Great flag; I was wrestling with this verification for a bit, and was still uneasy that it was doing the right thing. I’ll get this into a CLI test, and with that I should be able to cover this area pretty exhaustively.